sklearn/linear_model/base.py

```diff
@@ -238,16 +238,32 @@ def _predict_proba_lr(self, X):
         1. / (1. + np.exp(-self.decision_function(X)));
         multiclass is handled by normalizing that over all classes.
         """
+        from sklearn.linear_model.logistic import (
+            LogisticRegression, LogisticRegressionCV)
```

Comment:
That's pretty ugly. I would rather override `predict_proba` in `LogisticRegression`.

```
def predict_proba(self, X):
    if self.multiclass == "multinomial":
        [...]
    else:
        return self(LogisticRegression, self).predict_proba(X)
```